plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -2584,6 +2586,32 @@ def camel_to_snake_case(self, config):
 
         return new_config
 
+    def consolidate_output(self):
+        """
+        Bulk access point changes collect each output update in the response.
+
+        Parameters:
+            self (dict): A dictionary used to collect the execution results.
+
+        Returns:
+            dict: A dictionary containing the result of the access point update response.
+        """
+        each_result = {"changed": self.result["changed"],
+                       "response": self.result["response"].get("accesspoints_verify"),
+                       "ap_update_msg": self.result["ap_update_msg"]}
+        self.payload["consolidated_result"].append(each_result)
+        self.log("Each execution Result {0}".format(self.pprint(self.result)))
```

Comment:
Log the message after changing the result.. 
    self.result['changed'] = False
    self.log("Each execution Result {0}".format(self.pprint(self.result)))